front_end/panels/react_devtools/ReactDevToolsView.ts

```diff
@@ -109,6 +152,8 @@ export class ReactDevToolsViewImpl extends UI.View.SimpleView {
       bridge: this.bridge,
       store: this.store,
       theme: usingDarkTheme ? 'dark' : 'light',
+      canViewElementSourceFunction: () => true,
+      viewElementSourceFunction,
```

Comment:
There is also `fetchFileWithCaching` function, which we should propagate to DevTools, so DevTools can actually show symbolicated source of the component.

Without this, the `symbolicatedSource` will always be `null`. I will do this in [T190150458](https://www.internalfb.com/intern/tasks/?t=190150458).

Reply:
See [other comment](https://github.com/facebookexperimental/rn-chrome-devtools-frontend/pull/71#discussion_r1611865529) - I'm not convinced this is a net benefit. Happy for you to explore this though.